github/org_custom_roles.go

```diff
@@ -0,0 +1,38 @@
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OrginizationCustomRoles represents custom repository roles available in specified organization
```

Comment:
Please use complete sentences with punctuation for all godoc comments for auto-generated documentation.
Let's also add `Repo` to the name to better match the official documentation.

```suggestion
// OrginizationCustomRepoRoles represents custom repository roles available in specified organization.
```

Reply:
Organization* here and elsewhere.